### PR TITLE
bzip2: add examples

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bzip2.md [hunk 1]
```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test compressed file integrity:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
+
+- Verbose mode -- show the compression ratio for each file processed:
```
Comment (kbdharun) [inline pages/common/bzip2.md:22]:
```suggestion
- Show the compression ratio for each file processed in [v]erbose mode:
```

File: pages/common/bzip2.md [hunk 2]
```diff
@@ -14,3 +14,19 @@
...
+
+`bzip2 -v {{path/to/compressed_files.bz2}}`
+
+- Print help:
```
Comment (kbdharun) [inline pages/common/bzip2.md:26]:
```suggestion
- Display help:
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have some suggestions for this page.
- Update the documentation link with https://manned.org/bzip2 .
- Then add mnemonics [] to all command descriptions.
- Move the help example to last

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/bzip2.md [hunk 3]
```diff
@@ -14,3 +14,19 @@
...
+
+- Test compressed file integrity:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
+
+- Show the compression ratio for each file processed in [v]erbose mode:
+
+`bzip2 -v {{path/to/compressed_files.bz2}}`
+
+- Overwrite existing output files:
```
Comment (navarroaxel) [inline pages/common/bzip2.md:26]:
```suggestion
- Decompress a file overwriting existing files:
```

File: pages/common/bzip2.md [hunk 4]
```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test compressed file integrity:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
+
+- Show the compression ratio for each file processed in [v]erbose mode:
```
Comment (navarroaxel) [inline pages/common/bzip2.md:22]:
```suggestion
- Show the compression ratio for each file processed with detailed information:
```

File: pages/common/bzip2.md [hunk 5]
```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test compressed file integrity:
```
Comment (navarroaxel) [inline pages/common/bzip2.md:18]:
```suggestion
- Test integrity of each file inside the archive file:
```

File: pages/common/bzip2.md (hunk 4 above)
Comment (kbdharun) [reply pages/common/bzip2.md:22]: IMO, Both the versions are fine. What should we do?

File: pages/common/bzip2.md [hunk 6]
```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test integrity of each file inside the archive file:
```
Comment (kbdharun) [inline pages/common/bzip2.md:18]:
```suggestion
- Test the integrity of each file inside the archive file:
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/bzip2.md [hunk 7]
```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test the integrity of each file inside the archive file:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
```
Comment (EmilySeville7cfg) [inline pages/common/bzip2.md:20]:
```suggestion
`bzip2 --test {{path/to/compressed_file.bz2}}`
```

Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).

File: pages/common/bzip2.md [hunk 8]
```diff
@@ -14,3 +14,19 @@
...
+
+- Show the compression ratio for each file processed with detailed information:
+
+`bzip2 -v {{path/to/compressed_files.bz2}}`
```
Comment (EmilySeville7cfg) [inline pages/common/bzip2.md:24]:
```suggestion
`bzip2 --verbose {{path/to/compressed_files.bz2}}`
```

The same issue as above.

File: pages/common/bzip2.md [hunk 9]
```diff
@@ -14,3 +14,19 @@
...
+
+- Decompress a file overwriting existing files:
+
+`bzip2 -f {{path/to/compressed_file.bz2}}`
```
Comment (EmilySeville7cfg) [inline pages/common/bzip2.md:28]:
```suggestion
`bzip2 --force {{path/to/compressed_file.bz2}}`
```
The same issue as above.

File: pages/common/bzip2.md [hunk 10]
```diff
@@ -14,3 +14,19 @@
...
+
+- Display help:
+
+`bzip2 -h`
```
Comment (EmilySeville7cfg) [inline pages/common/bzip2.md:32]:
```suggestion
```

Warning: `-h` was not found in the [man page](https://manned.org/bzip2).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/bzip2.md (hunk 10 above)
Comment (navarroaxel) [reply pages/common/bzip2.md:32]: But the `-z` exists in the help info provided by the command.

Comment (EmilySeville7cfg) [reply pages/common/bzip2.md:32]: But this example referrers to `-h` not `-z`.

Comment (navarroaxel) [reply pages/common/bzip2.md:32]: I mean `-h`

Comment (EmilySeville7cfg) [reply pages/common/bzip2.md:32]: Can you provide a screenshot or link where I can find this option?

Comment (navarroaxel) [reply pages/common/bzip2.md:32]:
```
$ bzip2 -h
bzip2, a block-sorting file compressor.  Version 1.0.8, 13-Jul-2019.

   usage: bzip2 [flags and input files in any order]

   -h --help           print this message
   -d --decompress     force decompression
   -z --compress       force compression
   -k --keep           keep (don't delete) input files
   -f --force          overwrite existing output files
   -t --test           test compressed file integrity
   -c --stdout         output to standard out
   -q --quiet          suppress noncritical error messages
   -v --verbose        be verbose (a 2nd -v gives more)
   -L --license        display software version & license
   -V --version        display software version & license
   -s --small          use less memory (at most 2500k)
   -1 .. -9            set block size to 100k .. 900k
   --fast              alias for -1
   --best              alias for -9

   If invoked as `bzip2', default action is to compress.
              as `bunzip2',  default action is to decompress.
              as `bzcat', default action is to decompress to stdout.

   If no file names are given, bzip2 compresses or decompresses
   from standard input to standard output.  You can combine
   short flags, so `-v -4' means the same as -v4 or -4v, &c.

```

**Pull request merged**